aptos-move/aptos-vm-profiling/src/bins/run_move.rs

```diff
@@ -125,18 +127,27 @@ fn main() -> Result<()> {
         bail!("Wrong number of arguments.")
     }
 
+    let mut builder = SafeNativeBuilder::new(
+        LATEST_GAS_FEATURE_VERSION,
+        NativeGasParameters::zeros(),
+        MiscGasParameters::zeros(),
+        TimedFeatures::enable_all(),
+        Features::default(),
+    );
+
     let stdlib_addr = AccountAddress::from_hex_literal("0x1").unwrap();
-    let mut natives = all_natives(stdlib_addr, GasParameters::zeros());
+    let mut natives = all_natives(stdlib_addr, &mut builder);
     natives.push((
         stdlib_addr,
         Identifier::new("Test").unwrap(),
         Identifier::new("create_signer").unwrap(),
         make_native_create_signer(),
     ));
-    natives.extend(move_table_extension::table_natives(
-        stdlib_addr,
-        move_table_extension::GasParameters::zeros(),
-    ));
+    // FIXME: Find a way to re-enalbe this.
+    //natives.extend(aptos_table_natives::table_natives(
```

Comment:
why table extension can't be enabled here?

Reply:
Originally, we were using the table extension in third_party move, and that extension has special support in move_vm_test_utils that can be enabled via a cargo feature flag.

 In this PR, I forked the table extension in `aptos-move`, breaking that support. I'll fix this by making the aptos-core fork importing the relevant types (`NativeTableContext`,`TableResolver`) from the one. on third_party move, instead of making a copy of them. (We still need to fork the native functions themselves)